taxi/models.py

```diff
@@ -1,5 +1,6 @@
 from django.db import models
 from django.contrib.auth.models import AbstractUser
+from django.forms import forms
```

Comment:
The `ValidationError` should be imported from `django.core.exceptions` instead of `django.forms`. This is important for model validation. Update the import statement accordingly.